plugins/modules/vmware_vm_info.py

```diff
@@ -279,6 +279,11 @@ def get_all_virtual_machines(self):
 
             vm_folder = PyVmomi.get_vm_path(content=self.content, vm_name=vm)
             datacenter = get_parent_datacenter(vm)
+            datastore_url = list()
+            for entry in vm.config.datastoreUrl:
+                datastore_url.append({key: gettattr(entry, key)
+                    for key in dir(entry):
+                    if not key.startswith('_')})
```

Comment:
```suggestion
            datastore_url = list()
            for entry in vm.config.datastoreUrl:
                datastore_url.append({key: getattr(entry, key) for key in dir(entry) if not key.startswith('_')})
```

Reply:
Add a note in documentation stating that `datastore_url` is added in `1.18.0` version. Also, update the example section.